lib/SuperSelectField.d.ts

```diff
@@ -0,0 +1,51 @@
+declare module 'SuperSelectField' {
+    import {Component} from 'react';
+
+    export interface SelectFieldProps {
+        anchorOrigin?: any;
```

Comment:
Why everywhere types are any?
https://github.com/KnisterPeter/react-to-typescript-definitions/issues/439

Reply:
and this file should be in ignore to :)

Reply:
Speaking to yourself ? ^^

Reply:
Just info :)